### ISSUE TITLE
Harrass System: Players in Populated FOBs can be assigned both Attack and Pursuit AI objectives?

### ISSUE THREAD

**Issue by dijksterhuis** [Savage-Game-Design/Paradigm#5]
TL;DR -- The [server/functions/harass/fn_harass_create_squads.sqf](https://github.com/Savage-Game-Design/Paradigm/blob/development/server/functions/harass/fn_harass_create_squads.sqf) script reads like players inside populated FOBs shouldn't have tracker teams sent to them, but variables may be assigned in the wrong order, meaning FOBs do have tracker AI assigned.

Unsure if this is by design / intention (hacking the assignment order to also assign tracker teams to the FOB for higher AI count) or if it's just no-one has noticed the possible out of order assignment.

If the former, some clarification that it is intended would be appreciated.

If the latter, should be a simple fix and happy to push a PR.

-----

### 1. Find players to send tracker teams towards

`_friendlyPlayersToHarass` variable based on the original `_harrasablePlayers` variable defined [here](https://github.com/Savage-Game-Design/Paradigm/blob/development/server/functions/harass/fn_harass_create_squads.sqf#L28-L37)

```
// server/functions/harass/fn_harass_create_squads.sqf#L70

// Find players that aren't harassed enough.
private _friendlyPlayersToHarass = _harassablePlayers select {_x getVariable "harass_level" < 1};
```

### 2. Send attacks towards populated FOBs

```
// server/functions/harass/fn_harass_create_squads.sqf#L73-L91

// Keep occupied FOBs harassed
private _playersToRemove = [];
{
    // ...
    _playersToRemove append _players;
    // logic where each populated FOB has Attack AI obj assigned
} forEach para_g_bases;
```

### 3. Exclude players in populated FOBs from the list of players for tracker teams ....

**POTENTIAL BUG** modify `_harassablePlayers` -- this variable is never used after this point
```
// server/functions/harass/fn_harass_create_squads.sqf#L93

// prune the list of harrassable players to exclqude those inside FOBs under attack
private _harassablePlayers = _harassablePlayers - _playersToRemove;
```

### 4. Assign tracker teams to players who should be harrassed

Uses  `_friendlyPlayersToHarass` variable, defined before the FOB attack logic and based on the original `_harrasablePlayers` variable defined [here](https://github.com/Savage-Game-Design/Paradigm/blob/development/server/functions/harass/fn_harass_create_squads.sqf#L28-L37)

The tracker team / pursuit AI logic includes players in a populated FOB.

```
// server/functions/harass/fn_harass_create_squads.sqf#L97-L148

while {!(_friendlyPlayersToHarass isEqualTo []) && count _friendlyPlayersToHarass != _lastPlayersToHarassLength} do {
    _lastPlayersToHarassLength = count _friendlyPlayersToHarass;
    private _target = selectRandom _friendlyPlayersToHarass;

     // logic stuff
};
```
